Content.Server/GameObjects/Components/GUI/ServerHandsComponent.cs

```diff
@@ -459,6 +469,28 @@ public bool ThrowItem()
             return false;
         }
 
+        public void StartPulling(PullableComponent pullable)
+        {
+            if (isPulling)
+            {
+                (_pulledObject.Controller as PullController).StopPull();
```

Comment:
These should be hard casts `((PullController)_pulledObject.Controller)` because they will throw an null exception otherwise anyways.